framework/src/Volo.Abp.GlobalFeatures/Volo/Abp/GlobalFeatures/GlobalFeatureHelper.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Volo.Abp.Reflection;
+
+namespace Volo.Abp.GlobalFeatures
+{
+    public static class GlobalFeatureHelper
+    {
+        public static bool IsGlobalFeatureEnabled(Type controllerType, out RequiresGlobalFeatureAttribute attribute)
```

Comment:
```suggestion
        public static bool IsGlobalFeatureEnabled(Type type, out RequiresGlobalFeatureAttribute attribute)
```

controller is not a good naming in this layer.